src/Compilers/CSharp/Portable/Symbols/SignatureOnlyMethodSymbol.cs

```diff
@@ -22,11 +22,9 @@ internal sealed class SignatureOnlyMethodSymbol : MethodSymbol
         private readonly ImmutableArray<ParameterSymbol> _parameters;
         private readonly RefKind _refKind;
         private readonly TypeSymbol _returnType;
-        private readonly ImmutableArray<CustomModifier> _returnTypeCustomModifiers;
-        private readonly ushort _countOfCustomModifiersPrecedingByRef;
```

Comment:
Do we need distinct `SignatureOnly` classes with and without custom modifiers? Is it to reduce the size of allocations, and if so, are these long-lived symbols?